package.json

```diff
@@ -56,7 +57,7 @@
 	},
 	"husky": {
 		"hooks": {
-			"pre-commit": "npm run build && git add \"**/*.js\""
+			"pre-commit": "npm run lint && npm run build && git add \"**/*.js\""
```

Comment:
Note: I think adding linting would help as we don't have any CI/CD that runs the checks